### PR TITLE
WIP: Basic signing integration test

### PR DESCRIPTION
**Pull request by aweiteka**

### CODE REVIEW

**Comment by aweiteka** [comment]
@baude any ideas on the test failures? Is there some type of environment setup I'm missing?

**Comment by rhatdan** [comment]
@aweiteka can you merge all of these pull requests together and then we can merge if it passes.

**Comment by rhatdan** [comment]
@aweiteka are you still working on this one?

**Comment by aweiteka** [comment]
Test failures due to old skopeo version (skopeo version 0.1.14-dev commit: e8ab84dda9590796d6fe991e7ea177f0f78bec0e) in fedora/24/atomic

- no support for docker-daemon
- no /etc/containers/registries.d directory

**Comment by aweiteka** [comment]
@baude could you let me know when the fedora/atomic vagrant box gets an update of skopeo?

**Comment by rhatdan** [comment]
@aweiteka Please Rebase.

File: Makefile [hunk 1]
```diff
@@ -95,3 +101,9 @@ install-openscap:
 .PHONY: vagrant-check
 vagrant-check:
 	BOX=$(BOX) sh ./vagrant.sh 
+
+.PHONY: build-skopeo
+build-skopeo:
+	rm -rf $(DESTDIR)/skopeo
+	git clone --depth 1 $(SKOPEO_SRC_REPO) -b $(SKOPEO_SRC_BRANCH) $(DESTDIR)/skopeo
+	make binary --directory $(DESTDIR)/skopeo -d
```
Comment (jlebon) [inline Makefile]: Would it be sufficient to just enable updates-testing and install from there? You can add extra repos in the `.redhat-ci.yml` file (see [sample](https://github.com/jlebon/redhat-ci/blob/master/sample.redhat-ci.yml#L65)). For CentOS, you can use the [continuous RPMs](https://wiki.centos.org/SpecialInterestGroup/Atomic/Devel).

Comment (aweiteka) [inline Makefile]: This was stub'd out when we had dependencies on unreleased skopeo features. I'm not calling this in make test but left it in for local dev convenience. Not sure if it's useful or not.

**Comment by rhatdan** [comment]
@aweiteka You still want this merged.  I am creating atomic-1.14 today.

**Comment by aweiteka** [comment]
> @aweiteka You still want this merged. I am creating atomic-1.14 today.

It's a test so it's not something that needs to get into a release. Regardless, need to get this test passing.

**Comment by rhatdan** [comment]
@aweiteka What should we do with this. Trying to cleanup pull requests.

**Pull request closed without merging**